lib/internal/cluster/master.js

```diff
@@ -70,9 +70,9 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
-  const hasDebugArg = process.execArgv.some((argv) => {
-    return /^(--debug|--debug-brk)(=\d+)?$/.test(argv);
-  });
+  const hasDebugArg = process.execArgv.some(
+    (argv) => /^(--debug|--debug-brk)(=\d+)?$/.test(argv)
```

Comment:
shouldn't the `(argv) =>` be back where it use to be? is the linter forcing args to be pushed to the next line?